docs/src/main/asciidoc/_admin-guide/CLI_Recipes.adoc

```diff
@@ -190,14 +190,19 @@ Assume you have a host that is called "primary"
 
 [source,options="nowrap"]
 ----
-[domain@localhost:9990 /] /host=primary:read-config-as-xml
+[domain@localhost:9990 /] attachment display --operation=/host=primary:read-config-as-xml
 ----
 
 Just for the domain or standalone
 
 [source,options="nowrap"]
 ----
-[domain@localhost:9990 /] :read-config-as-xml
+[domain@localhost:9990 /] attachment display --operation=:read-config-as-xml
```

Comment:
According to the analysis document we'd actually need to use `read-config-as-xml-as-file`, right?